IABTechLab/uid2-self-serve-portal
# Ability to add Mobile App Ids in Client Side Integration

What Changed:
- Ability to add, edit and delete mobile app ids
- Ability to view app ids in a table
- Validation for the three types of mobile app ids
- Changed several folders/files/function names to be more generalized between app ids and domains
- Add app ids as field where needed
- Changed instructions where necessary
- Corresponding storybook files
- Updated version of braces npm package to fix vulnerability
- Changed `clientSideIntegration.tsx` to use new loader pattern

Test Plan:
- Add mobile app ids
- Add invalid app ids
- Add duplicate app ids
- Add app ids that already exist in the list
- Add app ids while deleting the existing ones
- Edit app id
- Edit app id to something invalid
- Delete a single app id
- Bulk delete app ids
- Search app ids
- Test all steps above for domains as well 
- Test add, edit, delete for key pair
- Check that rows are inserted into audit trails db when an action is performed
- Check new storybook files
